<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewViewVarDialog</class>
 <widget class="QDialog" name="NewViewVarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>变量名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描 述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数值类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>变量类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_VarName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_VarType">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>常量</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>枚举</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>字段</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>函数</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_VarValueType">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>Bool</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SByte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Byte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DateTime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Description">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_Confirm">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_Cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_ConstValue">
    <widget class="QLineEdit" name="lineEdit_Const_Value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_EnumValue">
    <widget class="QPushButton" name="pbt_Add_EnumVar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_enumValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_FieldValue">
    <widget class="QPushButton" name="pbt_Add_NewField">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbt_Add_DataArchiveField">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Field_Value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 值：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
